### PR TITLE
Feat/local multiplayer

### LINKED ISSUE #14: Implementar multijugador en red local

### PR DESCRIPTION
**Pull request by vacodoceo**

### CODE REVIEW

File: src/project.godot [hunk 1]
```diff
@@ -45,14 +45,6 @@ move_right={
  ]
 }
 
-[layer_names]
-
-2d_physics/layer_1="walls"
-2d_physics/layer_2="player"
-2d_physics/layer_3="obstacles"
-
```
Comment (AmbrosioV) [inline src/project.godot]: This has to stay

File: src/project.godot [hunk 2]
```diff
@@ -45,14 +45,6 @@ move_right={
...
 [rendering]
 
-quality/driver/driver_name="GLES2"
-vram_compression/import_etc=true
```
Comment (AmbrosioV) [inline src/project.godot]: Same, its needed

File: src/scenes/Player.tscn [hunk 3]
```diff
@@ -14,20 +14,10 @@ script = ExtResource( 2 )
 
 [node name="Sprite" type="Sprite" parent="."]
 texture = ExtResource( 1 )
-region_enabled = true
```
Comment (AmbrosioV) [inline src/scenes/Player.tscn:17]: I made the Sprite box smaller so it fits better

File: src/scenes/Player.tscn [hunk 4]
```diff
@@ -14,20 +14,10 @@ script = ExtResource( 2 )
...
-region_rect = Rect2( 49, 42, 35, 43 )
```
Comment (AmbrosioV) [inline src/scenes/Player.tscn]: I made the Sprite box smaller by enabling region, so it fits better

File: src/scenes/Player.tscn [hunk 5]
```diff
@@ -4,8 +4,8 @@
 [ext_resource path="res://scripts/Player.gd" type="Script" id=2]
 
 [sub_resource type="CapsuleShape2D" id=1]
-radius = 13.5838
-height = 0.0
+radius = 18.2147
+height = 13.2334
```
Comment (AmbrosioV) [inline src/scenes/Player.tscn]: Made the hitbox to fit only the lower part of the Sprite

File: src/scripts/Game.gd [hunk 6]
```diff
@@ -3,29 +3,38 @@ extends Node2D
...
-	var rock_group = get_tree().get_nodes_in_group("Rocks")
-	for rock in rock_group:
-		rock.connect("rock_collision", new_player, "_on_rock_collision")
-	var ice_group = get_tree().get_nodes_in_group("Ices")
-	for ice in ice_group:
-		ice.connect("ice_enter", new_player, "_on_ice_enter")
-		ice.connect("ice_exit", new_player, "_on_ice_exit")
-	var jump_group = get_tree().get_nodes_in_group("Jumps")
-	for jump in jump_group:
-		jump.connect("jump_exit", new_player, "_on_jump_exit")
```
Comment (AmbrosioV) [inline src/scripts/Game.gd:24]: This need to stay, the singal connections are necessary for gameplay

**Comment by domingo-valdes** [comment]
Everything working

**Pull request merged**